lib/ansible/utils/module_docs_fragments/rabbitmq.py

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2017 Ansible Project
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
```

Comment:
Could you please use - 

```
# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
# Copyright: (c) 2017, Ansible Project
``` 

instead of line 1 to 17

Reply:
Done :)